client/blockchain-service-db/migrations/00000000000001_create_pending_transactions/up.sql

```diff
@@ -0,0 +1,64 @@
+-- Create the pending_transactions table to coordinate txs across instances
+CREATE TABLE IF NOT EXISTS pending_transactions (
+  account_id BYTEA NOT NULL,
+  nonce BIGINT NOT NULL,
+  hash BYTEA NOT NULL,
+  call_scale BYTEA NOT NULL,
+  -- Full signed extrinsic bytes for re-subscription on restart
+  extrinsic_scale BYTEA NOT NULL,
+  -- Whether the node is actively watching this transaction via RPC subscription
+  watched BOOLEAN NOT NULL DEFAULT TRUE,
+  state TEXT NOT NULL CHECK (state IN (
+    'future','ready','broadcast','queued','sent','in_block','retracted','finalized','invalid','dropped','usurped','finality_timeout'
+  )),
+  creator_id TEXT NOT NULL,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT now(),
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT now(),
+  PRIMARY KEY (account_id, nonce)
+);
+
+CREATE INDEX IF NOT EXISTS pending_tx_hash_idx ON pending_transactions (hash);
+
+-- Notification trigger for new pending transactions (for follower instances)
+CREATE OR REPLACE FUNCTION notify_pending_tx_new() RETURNS trigger AS $$
+BEGIN
+  PERFORM pg_notify(
+    'pending_tx_new',
+    json_build_object(
+      'account_id', encode(NEW.account_id, 'hex'),
+      'nonce', NEW.nonce,
+      'hash', encode(NEW.hash, 'hex'),
+      'state', NEW.state,
+      'creator_id', NEW.creator_id
+    )::text
+  );
+  RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;
```

Comment:
Is this TODO refering to this notification?
https://github.com/Moonsong-Labs/storage-hub/blob/1fefd4a0486a56d118aa710224e4daeda721f6e9/client/blockchain-service-db/src/store.rs#L184